lib/auth/init.go

```
@@ -126,6 +130,34 @@ func Init(cfg InitConfig, seedConfig bool) (*AuthServer, *Identity, error) {
 	// and this is NOT the first time teleport starts on this machine
 	skipConfig := seedConfig && !firstStart
 
+	// set the cluster auth prerference, the first time read them from the config
+	// and create a resource on the backend, after that always read from the backend
+	if !skipConfig {
+		log.Infof("Initializing Cluster Authentication Preference: %+v", cfg.AuthPreference)
+		err = asrv.SetClusterAuthPreference(cfg.AuthPreference)
+		if err != nil {
+			return nil, nil, trace.Wrap(err)
+		}
+
+		log.Infof("Initializing Universal Second Factor Settings: %+v", cfg.U2F)
+		if cfg.U2F != nil {
+			err = asrv.SetUniversalSecondFactor(cfg.U2F)
```

The error is ignored